.ci/run_metrics_PR_ci.sh

```diff
@@ -102,6 +94,8 @@ check() {
 				sudo mkdir -p ${CHECKMETRICS_CONFIG_DEFDIR}
 				sudo cp ${CM_DEFAULT_DENSITY_CONFIG} ${CM_BASE_FILE}
 			fi
+		elif [ -n "${METRICS_CI_CLH}" ]; then
+			local CM_BASE_FILE="${CHECKMETRICS_CONFIG_DIR}/checkmetrics-json-clh.toml"
```

Comment:
Same here what about to rename to `checkmetrics-json-clh-baremetal.toml` so we can indetify just by name that is for baremetal.